redhat-developer/developers.redhat.com
# [MWES-3393] Clean temporary files from previous deployments as part of boot process

This PR makes a small update to the environment provisioning process to clean-up temporary files from the previous deployments.

### JIRA Issue Link
* https://issues.jboss.org/browse/MWES-3393

### Verification Process

* The build should go green
* [Log into the Drupal Container](https://github.com/redhat-developer/developers.redhat.com/tree/master/previews)
  * Ensure that files within the `/var/www/drupal/web/sites/default/files/js` directory are all showing the same date of creation using `ls`
  * Ensure that files within the `/var/www/drupal/web/sites/default/files/css` directory are all showing the same date of creation using `ls`
  * Ensure that files within the `/var/www/drupal/web/sites/default/files/externals` directory are all showing the same date of creation using `ls`
